control/bgp.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.3
+info:
+  title: BGP Control models
+  description: >-
+    All BGP control schemas
+  version: ^0.0.0
+components:
+  schemas:
+    BGP.Notification:
+      description: >-
+        A NOTIFICATION message is sent when an error is detected with the BGP session, such as hold timer expiring, misconfigured AS number 
+        or a BGP session reset is requested. This causes the BGP connection to close. Send user defined NOTIFICATIONs for list of specified 
+        BGP peers. A user can send IANA defined BGP NOTIFICATIONs according to https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml. 
+        If a user wants to send custom Error Code and Error Subcode the custom object should be used.
+      type: object
+      properties:
+        names:
+          description: >-
+            The names of BGP Peers to send NOTIFICATION to. If no name is specified then NOTIFICATION will be send to all BGP peers that matches with the
+            x-constraint.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - '/components/schemas/Device.Bgp/properties/name'
+          x-field-uid: 1
+        choice:
+          description: >-
```

Comment:
I would drop the first line (as we already mentioned what BGP notifications are) and start with "Each BGP NOTIFICATION message includes a Error Code field indicating the type of problem ..."

Reply:
Please replace "send" with "sent" wherever applicable and add additional newlines for readability. Right now, it looks like one big paragraph.

Reply:
"the choice name" -> "the choice value"

Reply:
"both these cases" -> "both of these cases"

Reply:
"CustomError can be used" -> "custom choice can be used"

Reply:
Question to @ashutshkumr : How to add new lines ? Looks like there is a way . Can we add that to otg doc.

Reply:
updated, but not seeing new lines in redocly